CHANGELOG.md

```diff
@@ -9,6 +9,10 @@ Rust SCIP bindings:
   the latest release v3.7.2.
   (https://github.com/sourcegraph/scip/pull/313)
 
+Go bindings:
+
+- IndexVisitor now accepts a context object and returns an error.
```

Comment:
```suggestion
- BREAKING CHANGE: IndexVisitor now accepts a context object and returns an error.
  When an error is returned, the visitor short-circuits.
```

Reply:
@varungandhi-src I think this was merged before the suggestion was committed.

Reply:
Errr... actually I just misread.